proto/src/main/proto/Rss.proto

```diff
@@ -335,6 +335,7 @@ message CheckServiceAvailableResponse {
 message AccessClusterRequest {
   string accessId = 1;
   repeated string tags = 2;
+  map<string, string> reservedData = 3;
```

Comment:
could we use the name `extraProperties`, or another more suitable name?

Reply:
extraProperties is ok.